### PR TITLE
update repository overview

### PR DESCRIPTION
**Pull request by surajkota**
**Description of your changes:**
Update repository overview to be more generic

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rrrkharse — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 ## Overview
 
-This repository contains configuration files and [overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) to deploy Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/).
+Kubeflow on AWS is an open source distribution of [Kubeflow](https://www.kubeflow.org/) that provides its own Kubeflow manifests to support integrations with various AWS managed services. Use Kubeflow on AWS to streamline data science tasks and build highly reliable, secure, and scalable machine learning systems with reduced operational overheads.
```
Comment (akartsky) [inline README.md:8]: `its own Kubeflow manifests`

- change this to `where we provide our own manifests`
- or just delete it

File: README.md [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
 
 ## Getting Started
 
-There are a number of deployment options for installing Kubeflow with AWS service integrations. For detailed instructions regarding available AWS-integrated deployment options, see the Deployment section on the [Kubeflow on AWS](https://awslabs.github.io/kubeflow-manifests/) website.
+There are a number of deployment options for installing Kubeflow with AWS service integrations. To get started with deploying Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/), see the [deployment section](https://awslabs.github.io/kubeflow-manifests/docs/deployment/) on the [Kubeflow on AWS](https://awslabs.github.io/kubeflow-manifests/) website.
```
Comment (akartsky) [inline README.md:12]: remove `deploying `

File: README.md (hunk 1 above)
Comment (surajkota) [reply README.md:8]: avoided the work we/you to make it more formal similar to website description. Reviewed it with Judy and Sree. lgt both

File: README.md (hunk 2 above)
Comment (surajkota) [reply README.md:12]: sentence reads strange without it - Kubeflow on EKS, Kubeflow on AWS

To get started with Kubeflow on Amazon EKS, see the deployment section on the Kubeflow on AWS website.
I made this change to include EKS specifically.

other option is - 
To get started, see the deployment section on the Kubeflow on AWS website.

wdyt?

File: README.md (hunk 1 above)
Comment (akartsky) [reply README.md:8]: oh okay

File: README.md (hunk 2 above)
Comment (akartsky) [reply README.md:12]: I think the sentence can constructed better 
we can do this later 
ok for now

**Review by akartsky — APPROVED** [review]

**Pull request merged**